graviti/dataframe/frame.py

```diff
@@ -194,6 +194,7 @@ def _from_pyarrow(cls: Type[_T], array: pa.StructArray, schema: pt.PortexType) -
         obj.schema = schema
         obj._columns = {}
         obj._column_names = []
+        obj.operations = None
```

Comment:
will it be better to set default value of `operations` to None, rather than set `operations` to None everywhere?